src/Actions/babylon.actionManager.ts

```diff
@@ -138,9 +143,8 @@
             return ActionManager._OnKeyUpTrigger;
         }
 
-        public static DragMovementThreshold = 10; // in pixels

```

Comment:
This is a breaking change I cannot validate..And this leads me to my other remark: Observable should remain generic. The hasPickMask function should not be there.

This is not technical but more philosophical :)